<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchesWindow</class>
 <widget class="QWidget" name="SearchesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Búsquedas Eficientes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="logoLB">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="searchLE">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
         <property name="placeholderText">
          <string>Criterio de búsqueda...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeSearchCB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">selection-background-color: rgb(52, 101, 164);</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
         <item>
          <property name="text">
           <string>Ninguna</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Búsqueda binaria</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>índices primarios</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Índices secundarios (Remitente)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>índices secundarios (Destinatario)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>índices secundarios (Booleana)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dispersión</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchPB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="mailTB">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FECHA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HORA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FROM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CCO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ASUNTO</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CONTENIDO</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="backMenuBT">
     <property name="text">
      <string>Regresar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
